scripts/preinstall-script.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+const setup =
+  '\nhttps://github.com/ViacomInc/data-point/blob/master/CONTRIBUTING.md#setup\n'
+
+if (process.env.npm_execpath.indexOf('yarn') === -1) {
+  console.error(
+    '\x1b[41m',
+    'Error: You must use yarn to install dependencies.\n\n',
+    'For help setting up this project please vist:',
+    '\x1b[0m',
+    '\x1b[36m',
+    '\n',
+    setup
+  )
+  process.exit(1)
+}
+
+console.warn(
+  '\x1b[36m%s\x1b[0m',
```

Comment:
@acatl I'm starting to think adding chalk as a dev dep is worthwhile. How many places are we handwriting color codes now?

Reply:
lets add it to the global dependency (root workspace) I only want us to be careful not to use it on data-point itself, if we can stick to that im 100% on board